<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Porta</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignLeft">
    <widget class="QSpinBox" name="spinPorta"/>
   </item>
   <item row="4" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="leArg"/>
   </item>
   <item row="5" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btEnv">
     <property name="text">
      <string>&amp;Enviar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btDes">
     <property name="text">
      <string>&amp;Desconectar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btCon">
     <property name="text">
      <string>&amp;Conectar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="leHost"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
